src/components/Pagination/Wrapper.jsx

```diff
@@ -0,0 +1,37 @@
+import { createElement as E, Component } from 'react'
+import T from 'prop-types'
+import { classNames } from '../../utils/'
+
+class PaginationWrapper extends Component {
+  render() {
+    const { node, children, className, activeLabel, ...rest } = this.props
+    return (
+      E(
+        node || 'nav',
+        {
+          role: 'navigation',
+          'aria-label': activeLabel,
```

Comment:
I'd call this prop `ariaLabel` and have a default value of 'pagination'.  `activeLabel` suggests some kind of state and there is none with these pagination components

Reply:
done